CONTRIBUTING.rst

```diff
@@ -11,6 +11,13 @@ Setting up the environment
 4. Install `pre-commit <https://pre-commit.com/>`_
 5. Run ``pre-commit install`` to install pre-commit hooks
 
+.. tip::
+  Many modern IDEs like PyCharm or VS Code will enable the poetry-managed virtualenv that is created in step 2 for you automatically.
```

Comment:
This is not a statement of endorsement for either, but I think these are the most well-known/commonly used IDEs and mentioning their names explicitly may help people who are really new to software development and might not recognize the term IDE.